t/ui/10-tests_overview.t

```diff
@@ -20,7 +20,7 @@ use lib "$FindBin::Bin/../lib", "$FindBin::Bin/../../external/os-autoinst-common
 use Date::Format 'time2str';
 use Test::Mojo;
 use Test::Warnings ':report_warnings';
-use OpenQA::Test::TimeLimit '20';
+use OpenQA::Test::TimeLimit '200';
```

Comment:
if you just need that for local testing you can also set the environment variable `OPENQA_TEST_TIMEOUT_DISABLE=1`